### PR TITLE
Blub balance

### PR DESCRIPTION
**Pull request by ghost**
Refer to issue #888
### Intent of your Pull Request
1. Blob burst time increased to 10 minutes.
2. Blob obj requirement for blob win increased to 700.
3. Blob expand to 75% chance, instead of 100%.
4. Anything else?

:cl: GROXIC
tweak: Blob burst time increased to 10 minutes.
tweak: Blob win requirement increased to 700 blobs.
tweak: Blob expand now 75% chance instead of 100%.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
what will increasing burst time to 10 minutes balance?

**Comment by MacHac** [comment]
:+1: 

@ShadowDeath6 It will balance the fact that right now, it's difficult to do anything meaningful to the station before bursting, since for all you know you have 1 minute to get to somewhere quiet.

File: code/game/gamemodes/blob/blob.dm [hunk 1]
```diff
@@ -69,7 +69,7 @@ var/list/blob_overmind_list = list()         //blob overmind list so we can keep
 	log_game("[blob.key] (ckey) has been selected as a Blob")
 	greet_blob(blobmind)
 	blob << "<span class='userdanger'>You feel very tired and bloated!  You don't have long before you burst!</span>"
-	spawn(600)
+	spawn(1000)
```
Comment (X-TheDark) [inline code/game/gamemodes/blob/blob.dm]: This is not 10 minutes...this is 100 seconds...needs to be changed to 6000

Comment (ghost) [inline code/game/gamemodes/blob/blob.dm]: Thank you! I'll fix it soon. Very tired.

**Comment by FluffySurvivor** [comment]
It's hilarious how nobody noticed how increasing the wincount does nothing but stall an unwinnable round for the crew. 

The blob already gets unbeatable at around 200 tiles unless a gamechanger like xray guns or mass bombing are brought to the table. Even these hard counters will be meaningless at around 600 tiles.

**Comment by Time-Green** [comment]
I think this is good.
More time is better since the crew has more time for a comeback like x-rays

**Comment by oisin100** [comment]
> Blob burst time increased to 10 minutes.

?!?!?!!!?!??

Thats enough time to maxcap bomb all the important stuff. Medbay, R&D And Genetics.

**Comment by AsV9** [comment]
I'd be up for this. It adds a new aspect to the blob mode..

**Comment by ghost** [comment]
Leave burst time as is.

@Fluffysurvivor this is intended if you just use lasers to damage 1-2 tiles.

As the blob gets bigger, you should stop using lasers, and use more advanced measures, like flamethrowers, flashbangs, or even bombs or plasma fires. In the meantime, xrays are the default hard counter to blobs-the rest of shit you pull off vs blobs is basically stalling for time until xrays come into the picture unless you catch it early enough.

Also with the reduced growth speed emitters ect can actually kill nodes. And if you upgrade, rip blob.

**Comment by ghost** [comment]
I need a common consensus on what the time before burst should be. I moved it from 1 minute, I think, to 10 minutes.

**Comment by ghost** [comment]
Burst has a minimum time of 1min and a max time of about 10.

I think you only raised the minimum time-I have seen blobs not burst for 8 minutes or so.

**Comment by ghost** [comment]
okay here are a couple more proposed balance changes

make fire instantly destroy blob cells. Like, kill 'em outright-any fire at all will instantly destroy normal tiles.

flashbangs, for some reason, only damage the first blobs in the wall, not the whole tile. Make flashbangs affect all blob tiles in view radius and it's far more balanced.

Remove blob node/blob core immunity to fire and make nodes take heavy damage when exposed to fire and make cores take medium damage when exposed to fire.

**Comment by AsV9** [comment]
What about giving blobs pre-burst some indication of how much time they have so people know if they have enough to do some sabotage or not.

**Comment by FluffySurvivor** [comment]
what will making the expand chance 75% balance ?

**Comment by ShadowDeath6** [comment]
Making cores die to fire is retarded
expand chance was lowered so blobs aren't completely immune to anything that isn't a fully upgraded emitter or xray laser

**Comment by FluffySurvivor** [comment]
> turning blob into farmville

Pls. Nobody will use expand anymore. It IS supposed to be the blob's most reliable tool. Up the cost to 7 and it's fine.

**Comment by ghost** [comment]
I'll keep working on this. Don't merge it.

**Comment by ghost** [comment]
no fluffy you're misunderstanding

blob expand chance was always 75% until the recent buff where it got 100% expand chance

expand refers to passive expansion not active ctrl-click

@FluffySurvivor 

also

@Groxic make fire not suck vs blobs

and perhaps on death of a node, halt resource regeneration for about 10-12 seconds so that a blob can't immediately plop down a new node

**Comment by ShadowDeath6** [comment]
> 10-12 seconds

Mate are you fucking serious
fire is already robust against blobs, don't make them bomb-tier

**Comment by X-TheDark** [comment]
I've never seen fire do much against blob as it's auto-expand takes care of any and all damage done to blob quickly after the fire subsides. This may change with the 75% expand chance, maybe fire will be robust and not need to be buffed.

**Comment by ghost** [comment]
fire is really weak vs blobs because it kills the crew trying to fight the blob and a dedicated fire breaches the hull so that the fire burns out @ShadowDeath6 

 also MERGE THIS ALREADY

**Comment by ShadowDeath6** [comment]
Merge this

**Comment by ghost** [comment]
@DavidEdwards HEY

**Comment by AsV9** [comment]
edit in new changelog pls

**Comment by ShadowDeath6** [comment]
Put blob burst time back to normal before merge pls

File: code/game/gamemodes/blob/blob.dm [hunk 2]
```diff
@@ -69,7 +69,7 @@ var/list/blob_overmind_list = list()         //blob overmind list so we can keep
 	log_game("[blob.key] (ckey) has been selected as a Blob")
 	greet_blob(blobmind)
 	blob << "<span class='userdanger'>You feel very tired and bloated!  You don't have long before you burst!</span>"
-	spawn(600)
+	spawn(6000)
```
Comment (ChangelingRain) [inline code/game/gamemodes/blob/blob.dm:72]: Funnily enough, this doesn't actually get called unless a blob tries to burst in space.

Comment (X-TheDark) [inline code/game/gamemodes/blob/blob.dm:72]: According to the code (at least on our codebase), it does call the burst_blob proc to burst the blob. The space check-warn thing is done in there. Unless I'm missing something.

**Comment by ghost** [comment]
Don't merge, updating sometime tonight.

**Comment by ShadowDeath6** [comment]
Merg wen
Edit: actually no don't merge until the blob timer is decreased

**Comment by oisin100** [comment]
Is this merge ready?

**Comment by ghost** [comment]
Alright, that's fucking enough

It's sat for a month and blob is sSTILL overpowered

Merge this and if anything else needs to be done do it in a future PR @oisin100

**Pull request merged**